extensions/graphql/src/__tests__/acceptance/graphql-middleware.acceptance.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/graphql
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {createRestAppClient, givenHttpServerConfig} from '@loopback/testlab';
```

Comment:
`createRestAppClient` <-- we should rename it, so we can use it with other API types without confusion.